doc/source/user_guide/sampling_wrappers.md

```diff
@@ -1,14 +1,14 @@
 (wrapper_guide)=
 # Sampling wrappers
-Sampling wrappers allow ArviZ to call PPLs in order to perform a limited
+Sampling wrappers allow ArviZ to call PPLs(Probabilistic programming library) in order to perform a limited
```

Comment:
definitely not a requirement and I don't even know if the role works with all themes or more specifically with ours so feel free to ignore, but that could be a good place to try out the abbr role: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-abbr